server/api/middlewares/authentication.js

```diff
@@ -1,20 +1,10 @@
 // @flow
-import httpErrors from 'http-errors';
 import JWT from 'jsonwebtoken';
 import { type Context } from 'koa';
-
 import { User, ApiKey } from '../../models';
+import { AuthenticationError } from '../../errors';
 
-type AuthOptions = {
-  require?: boolean,
```

Comment:
This was unused, if it's not required just don't add the middleware to that route I guess? 😄